torch/_refs/linalg/utils.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+from torch._prims.utils import (
+    check,
+    is_low_precision_dtype,
+    is_float_dtype,
+    is_complex_dtype,
+)
+
+
+def check_fp_or_complex(
+    dtype: torch.dtype, fn_name: str, allow_low_precision_dtypes: bool = False
```

Comment:
Seems like a function called "check floating point or complex" should allow any floating point or complex dtypes by default

Reply:
I would change it just to make it more readable, I understand why it would be convenient to leave this way, though